x-pack/plugins/cases/common/utils/user_actions.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function isCreateConnector(action?: string, actionFields?: string[]): boolean {
+  return action === 'create' && actionFields?.includes('connector') === true;
```

Comment:
I had to do the `=== true` to keep the values from interpreted as strings for some reason 🤷‍♂️

Reply:
You mean that if you do `return action === 'create' && actionFields?.includes('connector')` you will get a TS error? If that the case, the reason is happening is because `actionFields` can be `undefined`. You can also use:

- `return action === 'create' && !!actionFields?.includes('connector')`
- `return action === 'create' && actionFields != null  && actionFields?.includes('connector')`